content/en/docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -1,184 +1,232 @@
+
+* **oc cluster up** - A quick way to directly set up an OpenShift cluster on Fedora is to install a few required packages, then run **oc cluster up**, as described in [Start a local OpenShift all-in-one cluster](https://developer.fedoraproject.org/deployment/openshift/about.html).
+
+* **openshift-ansible** - Use ansible playbooks to set up OpenShift in a single-node cluster directly on a Fedora system. Ansible playbooks for running OpenShift are available from the [Kubespray](https://github.com/kubernetes-incubator/kubespray) Github site.
+
+The description here for running Kubernetes using kubeadm has been tested on Fedora and Fedora Atomic distributions. The resulting Kubernetes configuration has these attributes:
+
+* Tested on Fedora 28 Server and Fedora 28 Atomic Host Workstation
+* Single node Kubernetes cluster (continue to Kubernetes Multi-node Setup to add nodes)
+* Flannel networking
+* Pod address range of 10.244.0.0/16
+* For the single-node setup, master and node are on a single system, so the master is allowed to run pods
+
 
 ## Prerequisites
 
-1. You need 2 or more machines with Fedora installed. These can be either bare metal machines or virtual machines.
+Get and set up the latest Fedora or Fedora Atomic on any cloud, virtualization, or bare metal environment. To do that, you can obtain appropriate media and instructions for: 
 
-## Instructions
+* [Fedora Server Download](https://getfedora.org/en/server/download/) or
+* [Fedora Atomic Host Workstation Download](https://getfedora.org/en/workstation/download/)
 
-This is a getting started guide for Fedora.  It is a manual configuration so you understand all the underlying packages / services / ports, etc...
+Follow these steps to prepare the system:
 
-This guide will only get ONE node (previously minion) working.  Multiple nodes require a functional [networking configuration](/docs/concepts/cluster-administration/networking/) done outside of Kubernetes.  Although the additional Kubernetes configuration requirements should be obvious.
+1. Make sure that operating system software is up to date. On Fedora Atomic, type:
 
-The Kubernetes package provides a few services: kube-apiserver, kube-scheduler, kube-controller-manager, kubelet, kube-proxy.  These services are managed by systemd and the configuration resides in a central location: `/etc/kubernetes`.  We will break the services up between the hosts.  The first host, fed-master, will be the Kubernetes master.  This host will run the kube-apiserver, kube-controller-manager, and kube-scheduler.  In addition, the master will also run _etcd_ (not needed if _etcd_ runs on a different host but this guide assumes that _etcd_ and Kubernetes master run on the same host).  The remaining host, fed-node will be the node and run kubelet, proxy and docker.
+    <pre><tt># <b>atomic host upgrade</b>
```

Comment:
code fences would be more in line with the rest of the repo, but totally up to you